iotit.go

```diff
@@ -45,7 +45,7 @@ GLOBAL OPTIONS:
 var Version string
 
 // Env string came from linker
-var Env string
+var Env string = "dev"
```

Comment:
__[golint]__
* [golint] should omit type string from declaration of var Env; it will be inferred from the right-hand side

<!-- Comment by [SideCI](https://sideci.com) -->